contracts/src/protocol/OmniPortal.sol

```diff
@@ -102,6 +106,8 @@ contract OmniPortal is IOmniPortal, IOmniPortalAdmin, OwnableUpgradeable, OmniPo
         require(gasLimit <= XMSG_MAX_GAS_LIMIT, "OmniPortal: gasLimit too high");
         require(gasLimit >= XMSG_MIN_GAS_LIMIT, "OmniPortal: gasLimit too low");
         require(destChainId != chainId, "OmniPortal: no same-chain xcall");
+        require(destChainId != _BROADCAST_CHAIN_ID, "OmniPortal: no broadcast xcall");
+        require(to != _VIRUTAL_PORTAL_ADDRESS, "OmniPortal: no portal xcall");
```

Comment:
If we have reason to enable portal -> portal broadcasts xcalls in future, we can move these checks to external function.

Reply:
Is this check required? Seems to me this require statement will filter out calls that are a subset of lines 260-70

Reply:
This is saying that you can't make those calls from some evm (only the consensus chain can make those calls)